package.json

```diff
@@ -26,8 +30,7 @@
     "vue-apexcharts": "1.4.0",
     "vue-loading-overlay": "^3.2.0",
     "vue-router": "^3.0.3",
-    "vuex": "^3.0.1",
-    "websocket": "1.0.30"
+    "vuex": "^3.0.1"
```

Comment:
We don't need manually install `websocket`, specifying `node` version here is good enough to fix the issue when deploying to heroku. @lsankar4033